wp-seo-main.php

```diff
@@ -11,7 +11,7 @@
 }
 
 /**
- * @internal Nobody should be able to overrule the real version number as this can cause serious issues
+ * @note Nobody should be able to overrule the real version number as this can cause serious issues
```

Comment:
Sounds like a bug in PHPStorm and should be fixed there, not in our code. @maartenba might be able to clarify.